site/en/docs/extensions/mv3/intro/index.md

```diff
@@ -10,40 +10,26 @@ updated: 2022-09-28
 
 {% Partial 'extensions/mv3-support.md' %}
 
-This site introduces Manifest V3, short for Manifest Version 3, which is the latest iteration of the Chrome extension platform. It shares the background and reasons for introducing Manifest V3 and the vision for the platform's future, along with resources on how to migrate.
+Manifest Version 3 (Manifest V3) is the latest iteration of the Chrome extension platform. This page outlines the changes in Manifest V3 and provides migration resources.
 
-## What is a manifest? {: #what-is-a-manifest }
+An extension [manifest][doc-manifest] gives the browser information about an extension's capabilities and the files it uses. The features available for extensions to use are defined by the current [manifest version][manifest-version]. Manifest V3 introduces enhancements to extension security, privacy, and performance, and allows extensions to use open web technologies such as service workers and promises.
 
-An extension [manifest][doc-manifest] gives the browser information about the extension, such as the most important files and the capabilities the extension might use. The extension platform features change when there's a new [manifest version][manifest-version].
+The Chrome Web Store no longer accepts new Manifest V2 extensions, and Manifest V2 will be [phased out][mv2-sunset] in 2024. We strongly recommend migrating your extensions to Manifest V3 as soon as possible.
```

Comment:
```suggestion
The Chrome Web Store no longer accepts new Manifest V2 extensions. Watch the [Manifest V2 support timeline][mv2-sunset] for details. We strongly recommend migrating your extensions to Manifest V3 as soon as possible.
```

Reply:
The date should not have even been in what you started with since we've pushed back removal a few times already.  This wording future-proofs this page.